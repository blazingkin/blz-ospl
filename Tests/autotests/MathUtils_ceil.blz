:main
    print(ceil(20.123456))
    print(ceil(20))
    print(ceil(-4))
end