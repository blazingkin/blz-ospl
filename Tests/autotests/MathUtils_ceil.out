21.000000
21
-3
